dissect/util/feature.py

```diff
@@ -0,0 +1,66 @@
+import functools
+import os
+from enum import Enum
+from typing import Callable, Optional
+
+
+# Register feature flags in a central place to avoid chaos
+class Feature(Enum):
```

Comment:
If you make it a `Flag` you can OR features together.

Reply:
But then I have to override the __or__/__ror__ myself because the values are strings, or I need to map the strings to the flag first. All of which mean extra code, which might be unnecessary. Are you sure you want this?

Reply:
Hm you're right. While it feels more correct, it's more effort to achieve the same as you currently do with the split on `/`.